cluster/images/etcd/migrate-if-needed.sh

```diff
@@ -100,24 +100,24 @@ fi
 # Returns 0 if etcd was successfully started, non-0 otherwise.
 start_etcd() {
   # Use random ports, so that apiserver cannot connect to etcd.
-  ETCD_PORT=18629
-  ETCD_PEER_PORT=18630
+  ETCD_PORT=2379
```

Comment:
If possible, we want to avoid changing this, to not expose etcd for apiserver during migration (though if you need it for testing purposes, I'm obviously fine with this change).